Matrix.SynapseInterop.Common.MatrixUtils/EventFormatter.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Matrix.SynapseInterop.Database.SynapseModels;
+using Newtonsoft.Json.Linq;
+
+namespace Matrix.SynapseInterop.Common.MatrixUtils
+{
+    public static class EventFormatter
+    {
+        public static JObject ToClientServerFormat(this EventJson ev)
```

Comment:
Presumably this isn't going to handle v1 formats like synapse currently does?

Reply:
v1 as in c2s v1 or v1 as in something else?

Reply:
I thought appservices still had the "old" format for events which had things like a "user_id" instead of "sender" for backwards compat reasons.

Reply:
Spec says they receive events in the client server format. Sorry ancient appservices, you're going to need to update eventually.